docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+     - 0.50
+   * - Simple Syrup B
+     - Simple Syrup, Sweetener
+     - 0.45
+     - 0.55
+
+We now create margaritas by mixing together a subset of ingredients and blending for some time.
+The basic ingredients are labeled as described in the "Ingredients Data" table, and the simple syrups are given a "simple syrup" label.
+In addition to the ingredient mass fractions, we record the blending time, in seconds.
+The margaritas are judged on an ultra-rigorous "tastiness" scale, which ranges from 0.0 to 10.0
+The table below shows two examples.
+
+.. list-table:: Margarita Data
+   :widths: 35 25 40 35 35 35 35 25 25 25
+   :header-rows: 1
+
+   * - Name
+     - Tastiness
+     - Blend Time (s)
+     - Simple Syrup A Quantity
+     - Simple Syrup B Quantity
+     - Sugar Quantity
+     - Lime Juice Quantity
+     - Triple Sec Quantity
+     - Tequila Quantity
+     - Ice Quantity
+   * - Margarita A
+     - 6.3 +/- 0.1
+     - 8.0
+     - 0.20
+     - 0.0
+     - 0.0
+     - 0.15
+     - 0.0
+     - 0.25
+     - 0.40
+   * - Margarita B
+     - 5.4 +/- 0.1
+     - 12.0
+     - 0.0
+     - 0.15
+     - 0.0
+     - 0.10
+     - 0.10
+     - 0.30
+     - 0.35
+
+Ingesting Data
+--------------
+
+.. Warning:: Ingesting data to GEMD is best done with additional tooling to automate the process.
+
+
+Although we will not describe every step of the data ingestion process, we will highlight several data objects and their inter-connections.
+Perhaps most crucial are the `attribute templates` that correspond to the parameters of the blending process and the properties of the ingredients/formulations.
+The name, bounds, and units on these templates will later be matched to descriptors in the AI Engine.
+Make sure to set the bounds wide enough to encompass all anticipated use cases of the templates.
+
+.. note:: In the code examples that follow, imports are assumed to be cumulative.
+    Once an import has been demonstrated, we do not include it in subsequent code blocks.
+
+.. code-block:: python
+
+    from citrine.resources.parameter_template import ParameterTemplate
+    from citrine.resources.property_template import PropertyTemplate
+    from gemd.entity.bounds import RealBounds
+
+    # Assume we have already created a project and dataset, named "project" and "dataset."
+    # Though not demonstrated here, it is useful to give templates custom ids so that they can be easily retrieved.
+    blend_time_template = dataset.parameter_templates.register(
+        ParameterTemplate("blend time", bounds=RealBounds(0, 60.0, "s"))
+    )
+    price_template = dataset.property_templates.register(
+        PropertyTemplate("price", bounds=RealBounds(0, 100, "1/kg"))
+    )
+    sucrose_fraction_template = dataset.property_templates.register(
+        PropertyTemplate("sucrose fraction", bounds=RealBounds(0, 1, ""))
+    )
+    tastiness_template = dataset.property_templates.register(
+        PropertyTemplate("tastiness", bounds=RealBounds(0, 10, ""))
+    )
```

Comment:
Given that you registering these, would it be better to assign UIDs and use LinkByUID for reference, rather than object references?

Reply:
yes, it would